internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -345,6 +345,43 @@ func (provisioner *KopsProvisioner) ProvisionCluster(cluster *model.Cluster, aws
 			DeployNamespace: "kube-system",
 		},
 	}
+
+	_, err = k8sClient.Clientset.CoreV1().Namespaces().Get(ctx, "pgbouncer", metav1.GetOptions{})
+	if k8sErrors.IsNotFound(err) {
+		logger.Info("namespace resource does not exist, will create it...")
```

Comment:
I think it makes more sense to create the namespace with a API call instead of deploying a manifest. We already do this [here](https://github.com/mattermost/mattermost-cloud/blob/600a375c7076d9f0a17280413d23f8b276783873/k8s/namespaces.go#L33) 1/5 on that.

Reply:
Thanks @stylianosrigas for your suggestion, I have updated it.